DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Features/PointFeatureToSpeckleConverter.cs

```diff
@@ -1,37 +1,20 @@
 using ArcGIS.Core.Geometry;
 using Speckle.Converters.Common.Objects;
 using Speckle.Core.Models;
-using Speckle.Converters.Common;
-using ArcGIS.Desktop.Mapping;
 
 namespace Speckle.Converters.ArcGIS3.Features;
 
-[NameAndRankValue(nameof(MapPoint), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class PointFeatureToSpeckleConverter : IHostObjectToSpeckleConversion, IRawConversion<MapPoint, Base>
+public class PointFeatureToSpeckleConverter : IRawConversion<MapPoint, List<Base>>
 {
-  private readonly IConversionContextStack<Map, Unit> _contextStack;
+  private readonly IRawConversion<MapPoint, SOG.Point> _pointConverter;
 
-  public PointFeatureToSpeckleConverter(IConversionContextStack<Map, Unit> contextStack)
+  public PointFeatureToSpeckleConverter(IRawConversion<MapPoint, SOG.Point> pointConverter)
   {
-    _contextStack = contextStack;
+    _pointConverter = pointConverter;
   }
 
-  public Base Convert(object target) => RawConvert((MapPoint)target);
-
-  public Base RawConvert(MapPoint target)
+  public List<Base> RawConvert(MapPoint target)
   {
-    if (
-      GeometryEngine.Instance.Project(target, _contextStack.Current.Document.SpatialReference)
-      is not MapPoint reprojectedPt
-    )
-    {
-      throw new SpeckleConversionException(
-        $"Conversion to Spatial Reference {_contextStack.Current.Document.SpatialReference} failed"
-      );
-    }
-    List<Base> geometry =
-      new() { new SOG.Point(reprojectedPt.X, reprojectedPt.Y, reprojectedPt.Z, _contextStack.Current.SpeckleUnits) };
-
-    return geometry[0];
+    return new() { _pointConverter.RawConvert(target) };
```

Comment:
What is this doing other than returning a list with 1 single point? You might as well do that manually in `GeometryToSpeckleBase` :)

Reply:
this is done to keep consistency in the ToSpeckleGeometry converters. Ideally, at some point we want a new Interface which can be used to ResolveType (similar to IHostObjectToSpeckle) but return List in the end